src/cmd/tests/load_config_test.rs

```diff
@@ -234,6 +239,10 @@ fn test_load_standalone_example_config() {
                 remote_write: Some(Default::default()),
                 ..Default::default()
             },
+            http: HttpOptions {
+                cors_allowed_origins: vec!["https://example.com".to_owned()],
```

Comment:
Keep consistent with the above.
```suggestion
                cors_allowed_origins: vec!["https://example.com".to_string()],
```